app/grandchallenge/evaluation/admin.py

```diff
@@ -112,7 +112,16 @@ def challenge_forge_json(obj):
 def reevaluate_submissions(modeladmin, request, queryset):
     """Creates a new evaluation for an existing submission"""
     for submission in queryset:
-        submission.create_evaluation()
+        if submission.phase.inputs.exists():
```

Comment:
Is there an API difference here - the phase `inputs` are ComponentInterfaces, whereas for ComponentJobs `inputs` are ComponentInterfaceValues?

Reply:
Yes. This is about whether or not to allow creating a new Evaluation (ComponentJob). In the admin we can only allow this if the phase has no input sockets  (ie. ComponentInterfaces) configured. If it does, re-evaluation needs to happen through the challenge UI because we'll need to also provide the values for these input sockets.

Reply:
Does my comment address you question / concern?